From: Matthias Klumpp <matthias@tenstral.net>
Date: Tue, 21 May 2024 02:33:55 +0200
Subject: metainfo: Mark PK as compulsory for most desktops

Ideally, AppStream needs a way to just mark a component as critical for
the system to function and prevent its removal. But this is okay as a
temporary fix.

Resolves: #759

Origin: upstream, https://github.com/PackageKit/PackageKit/commit/d8f84f7e0a017e0b4df1e08a2a6be832c9fb3611
---
 data/org.freedesktop.packagekit.metainfo.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/data/org.freedesktop.packagekit.metainfo.xml b/data/org.freedesktop.packagekit.metainfo.xml
index f779b9d..5a8ad7a 100644
--- a/data/org.freedesktop.packagekit.metainfo.xml
+++ b/data/org.freedesktop.packagekit.metainfo.xml
@@ -27,6 +27,16 @@
   <url type="homepage">https://www.freedesktop.org/software/PackageKit</url>
 
   <launchable type="service">packagekit.service</launchable>
+
+  <compulsory_for_desktop>Cinnamon</compulsory_for_desktop>
+  <compulsory_for_desktop>GNOME</compulsory_for_desktop>
+  <compulsory_for_desktop>KDE</compulsory_for_desktop>
+  <compulsory_for_desktop>LXQt</compulsory_for_desktop>
+  <compulsory_for_desktop>MATE</compulsory_for_desktop>
+  <compulsory_for_desktop>Pantheon</compulsory_for_desktop>
+  <compulsory_for_desktop>Unity</compulsory_for_desktop>
+  <compulsory_for_desktop>XFCE</compulsory_for_desktop>
+
   <categories>
     <category>System</category>
   </categories>
